views/api/google-drive.pug

```diff
@@ -0,0 +1,13 @@
+extends ../layout
+
+block content
+  .pb-2.mt-2.mb-4.border-bottom
+    h2
+      i.fab.fa-google.fa-sm
+      | Google Drive API List Of Files
+    for file in files
+      li(title=file.id)
+        =file.name
+
+  script.
```

Comment:
Is this a debug script that needs to be removed?

Reply:
I've added it because "founding how to debug with pug" a bit confusing... It could be useful, but sure I can remove it.

Reply:
Removed.